src/psyclone/tests/psyGen_test.py

```diff
@@ -253,6 +256,32 @@ def apply(self, node, valid2: int = 1):
     assert options['valid2'].typename == "int"
 
 
+def test_transformation_get_valid_options_no_sphinx():
+    '''Test that the get_valid_options method behaves in the expected
+    way when sphinx isn't available.'''
+    # Test that importing stringify_annotations works without sphinx.
+    # Trick the import into thinking sphinx.util.typing is unavailable
+    with patch.dict(sys.modules, {'sphinx.util.typing': None}):
+        # Unload the previously imported Transformation class
+        # pylint: disable=import-outside-toplevel
+        del sys.modules['psyclone.psyGen']
```

Comment:
Aha! I've often had this problem in the past (wanting to patch a class that has already been imported) so it's really good to know how it can be done.